.vib/wordpress/cypress/cypress/fixtures/posts.json

```diff
@@ -0,0 +1,3 @@
+{
+  "title": "Hello world!"
```

Comment:
This should be an array of `posts`, in which one of them has that property.

Reply:
Good point, on my way!